modules/dag-l1/src/main/scala/org/tessellation/dag/l1/Main.scala

```diff
@@ -63,14 +65,20 @@ object Main
         sdkResources.client,
         currencyPathPrefix = "dag"
       )
+      maybeMajorityPeerIds =
```

Comment:
We don't perform this logic for core so that's why we need to implement it in every module other than core right (rather than in TessellationIOApp)? Maybe we can extract it to a function given that it seems like the same logic repeating in several places.

Reply:
If we're not using it in core module, why we need to (can) load the priority seedlist that makes no effect there? 🤔